network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -438,6 +439,27 @@ class ParticipateHighlights2(ParticipateHighlightsBase):
     )
 
 
+class PartnerLogos(WagtailOrderable):
+    page = ParentalKey(
+        'wagtailpages.Homepage',
```

Comment:
Let's stick with class casing:
```suggestion
        'wagtailpages.HomePage',
```

Reply:
actually, no. It really is Homepage - ignore this =D